software/controller/lib/hal/gpio.cpp

```diff
@@ -26,28 +26,6 @@ Reference abbreviations [RM], [DS], etc are defined in hal/README.md.
 
 namespace GPIO {
 
-// General Purpose I/O
```

Comment:
Putting this register structure into the `cpp` was very deliberate. We want to have the interface for doing GPIO be abstract, and not have the user have to know how that actually is under the hood. As, for example, to make it easier to change out processor architecture.